src/components/views/buildSomething/BuildSomething.js

```diff
@@ -31,7 +40,10 @@ class BuildSomething extends Component {
                 LiskHunt has the aim to collect all the resources, categorize
                 and advertise.<br />
                 You can take inspiration on the{' '}
-                <a onClick={goResourcesList.bind(this)} className="underlined">
+                <a
```

Comment:
I think that the " ' " broke Github :D Perhaps we should replace it with `&apos;` ?

Reply:
( on line 26 )

Reply:
Haha do you mean {" "}? This happens because of the prettier i think?

Reply:
I don't think that's the reason, I think the reason is the non escaped " ' " on line 26... let's see what happens if we escape that char :)